types/framework/entities/chatMessage.d.ts

```diff
@@ -211,7 +214,7 @@ declare class ChatMessage extends Entity<ChatMessage.Data> {
    * @remarks
    * Returns Actor.data.data
    */
-  getRollData(): any; // Actor.data.data
+  getRollData<A extends Actor = Actor>(): A['data']['data']; // Actor.data.data
```

Comment:
Is this really a good thing to do? I mean having a generic on method level is basically only as safe as a cast. If the data that is returned by this method is already determined on instantiation of the class, the whole class should be generic.

Reply:
I reverted this for now.